packages/amplify-graphql-transformer-core/src/transformation/transform.ts

```diff
@@ -392,11 +392,7 @@ export class GraphQLTransform {
     try {
       sandboxNode.run(overrideCode, overrideFilePath).override(appsyncResourceObj, projectInfo);
     } catch (err) {
-      throw new AmplifyError('InvalidOverrideError', {
-        message: 'Executing overrides failed.',
```

Comment:
As mentioned at the top-level, let's double-click that this is still being caught up-top and transformed into a well-formed AmplifyError.